CHANGELOG.md

```diff
@@ -1,6 +1,46 @@
 # Unreleased
 
-Nothing Yet!
+# Version 0.22.0 (2024-08-28)
+
+This patch release fixes several bugs and provides some nice quality of life improvements. These fixes are largely related to our installers, shell, powershell, and homebrew as well as managing some papercuts in the `dist init` command and error handling.
+
+## App-branded installer environment variables
+
+We generate installers for our users- and then our *users' users* use those installers. Those users don't know they are using `dist` and, if we do our job right, should never have to.
+
+While we had implemented some environment variables that enable users to control the behavior of installers, previously they were largely designed for internal use, and therefore namespaced with `DIST`. However, we quickly realized that this isn't suitable for having our users communicate with their users- so we've enabled the generation of app-branded installer customization environment variables- so that installer users can leverage environment variables that are branded to the application they are trying to install.
+
+Right now, the only customization we allow for installers is the install directory- so instead of `CARGO_DIST_FORCE_INSTALL_DIR`, you can tell your users to use `AXOLOTLSAY_INSTALL_DIR` (if your app is named `axolotlsay`). If you have a name with hypens or other characters, we normalize it for you, and you can find this value at the `install_dir_env_var` in the `dist-manifest.json` that is generated with each of your releases.
```

Comment:
I think you accidentally a word.

```suggestion
Right now, the only customization we allow for installers is the install directory- so instead of `CARGO_DIST_FORCE_INSTALL_DIR`, you can tell your users to use `AXOLOTLSAY_INSTALL_DIR` (if your app is named `axolotlsay`). If you have a name with hypens or other characters, we normalize it for you, and you can find this value at the `install_dir_env_var` field in the `dist-manifest.json` that is generated with each of your releases.
```